manifests/install.pp

```diff
@@ -13,4 +13,24 @@
       require => $pkg_req,
     }
   }
+
+  if $facts['os']['family'] in ['RedHat'] and $dns::globals::scl {
+    ['dig', 'nsupdate', 'rdnc'].each | $util | {
+      file { "/usr/bin/${util}":
```

Comment:
Should this also require `$pkg_req`? If the package somehow failed to install, the wrappers are rather useless.

Reply:
You are right - changed